src/containers/SignUp.js

```diff
@@ -0,0 +1,201 @@
+import React, { Component, Fragment } from 'react';
+import { Redirect } from 'react-router';
+import {
+  Form,
+  Grid,
+  Header,
+  Image,
+  List,
+  Message,
+  Segment,
+} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { withCookies, Cookies } from 'react-cookie';
+import axios from 'axios';
+
+import logoImage from '../assets/images/rovercode_logo.png';
```

Comment:
I think I'm going to re-factor this so that the logo is in more of a base template with the `Login`, `SignUp`, and `PasswordReset` (coming soon) as internal routes. I think it makes sense to mess with that in one place at that point.